autoapi/mappers/javascript.py

```diff
@@ -35,7 +35,7 @@ def read_file(self, path, **kwargs):
     def map(self, options=None):
         '''Trigger find of serialized sources and build objects'''
         for path, data in self.paths.items():
-            for item in data:
+            for item in (i for i in data if not u'files' in i):
```

Comment:
The lint error is about this, it should be: `for item in (i for i in data if u'files' not in i):`

Reply:
I follow your recommendation but have not way of checking this.
![2017-11-14](https://user-images.githubusercontent.com/14262513/32773686-48230c6a-c92a-11e7-8b11-2b9fb57cf45b.png)